paper.tex

```diff
@@ -238,7 +238,7 @@ \section*{Abstract}
 
 Lessons take significant effort to build,
 and even more effort to maintain.
-The collaborative code development methods pioneered by the open source community
+The collaborative code development methods pioneered by the open-source community
```

Comment:
A search for "open source community" returns about 11 times as many results on Symbol Hound as a search for the hyphenated "open-source community", so I suggest we stick to the former.  (Had to use Symbol Hound because Google throws away hyphens in searches, even when you quote the search string or hand-edit the URL. *sigh*)